package.json

```diff
@@ -256,6 +266,11 @@
         "command": "one.editor.cfg.setDefaultValues",
         "title": "ONE: Set Default Values",
         "category": "ONE"
+      },
+      {
+        "command": "one.editor.mpq.showFromDefaultExplorer",
```

Comment:
May I ask why the command name is 'show*', as far as I understand, it should be 'createMPQJson'...?

Reply:
Ahh. Sorry. I'll change it `createFromDefaultExplorer`.

Reply:
It was meant to show command related to mpq.